department-of-veterans-affairs/va.gov-cms
# VACMS-4415: Add 4 new fields to view.

## Description
See #4415

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/111835609-8bcae280-88cb-11eb-9bcf-eb604e67c69e.png)

## QA steps
As an administrator, go to `vha-health-services` and visually verify 4 new fields appear, per screenshot

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
